make-rules/common.mk

```diff
@@ -35,6 +35,16 @@ BUILD_STYLE ?= configure
 # Some build styles might want to set some defaults before prep.mk is included.
 -include $(WS_MAKE_RULES)/$(strip $(BUILD_STYLE))-defaults.mk
 
+SINGLE_PERL_VERSION ?= yes
```

Comment:
Maybe add some explanation about its purpose?

Reply:
Any suggestion about wording?  It is so obvious for me that it does not deserve (IMHO) any comment.  But if something is suggested I'll add it to make it clear(er) for others too.  Thanks.

Reply:
Something like:
Use only a single python/perl runtime for packages that don't provide python/perl libraries (but only use it as a runtime).

Reply:
Hmmmmm.  I think this is not entirely true.  For example `barman` provides python libraries (see `/usr/lib/python3.9/vendor-packages/barman/`).

Maybe something like:  _Build for single perl/python version only when you do not need more._  But this is something that is obvious when you read next three lines of code.

Reply:
The comment should be for anybody with only little package creation experiences.
The idea of a single perl/python runtime is: My package doesn't need more than one version. Either because its only using it as a runtime and/or I want to provide or maintain a single variant only.

Reply:
Yes, I agree.  The problem (or, actually, this is not a problem at all) is that somebody with little package creation experiences will probably never need to change `SINGLE_PERL_VERSION`/`SINGLE_PYTHON_VERSION` because defaults in most cases is what they need.  For example for gdb (PR #9568) nothing needs to be do and single python build will be selected automagically.

Anyway, I'll try to add some comment there...